rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -334,7 +334,23 @@ def test_find_uniswap_v2_lp_token_price(inquirer, ethereum_manager):
     address = '0xa2107FA5B38d9bbd2C461D6EDf11B11A50F6b974'
     identifier = ethaddress_to_identifier(address)
     inquirer.inject_evm_managers([(ChainID.ETHEREUM, ethereum_manager)])
-    price = inquirer.find_uniswap_v2_lp_price(EvmToken(identifier))
+    price = inquirer.find_lp_price(
+        token=EvmToken(identifier),
+        abi_name='UNISWAP_V2_LP',
+    )
+    assert price is not None
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
```

Comment:
If you mock the price I believe you never contact the inquirer, that the fixture returns a price before asking it